src/constants/action-types.js

```diff
@@ -127,9 +128,10 @@ const ActionTypes = keyMirror({
   REMOVE_NOTIFICATION: null,
 
   // uiState > export image
-  SET_RATIO: null,
-  SET_RESOLUTION: null,
-  TOGGLE_LEGEND: null,
+  // SET_RATIO: null,
+  // SET_RESOLUTION: null,
+  // TOGGLE_LEGEND: null,
```

Comment:
is this correct?

Reply:
these actions are deleted, all handled by one: `SET_EXPORT_IMAGE_SETTING`

Reply:
what about TOGGLE_LEGEND?